.gitignore

```diff
@@ -90,3 +90,6 @@ env
 
 # Zappa API deployment secrets
 zappa_settings.json
+
+database.dbml
+package*json
```

Comment:
😨 Good gracious John, thank you for catching that!! 

I had added those because during testing, right in the root of the repo, I had `npm install`'d the `dbdocs` tooling, which created those package files and a `node_modules` directory. It was causing my VSCode to go bonkers with all the files it created, so I stuck them in the `.gitignore` file -- never even considered the utter mayhem it would create for the apps in the repo.